test/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -53,14 +55,43 @@ public void commaListToSet() {
         Set<String> set = BridgeUtils.commaListToSet("a, b , c");
         assertEquals(Sets.newHashSet("a","b","c"), set);
         
+        set = BridgeUtils.commaListToSet("a,b,c");
+        assertEquals(Sets.newHashSet("a","b","c"), set);
+        
         set = BridgeUtils.commaListToSet("");
         assertEquals(Sets.newHashSet(), set);
         
         set = BridgeUtils.commaListToSet(null);
-        assertEquals(Sets.newHashSet(), set);
+        assertNotNull(set);
         
         set = BridgeUtils.commaListToSet(" a");
         assertEquals(Sets.newHashSet("a"), set);
+        
+        // Does not produce a null value.
+        set = BridgeUtils.commaListToSet("a,,b");
+        assertEquals(Sets.newHashSet("a","b"), set);
+    }
+    
+    @Test
+    public void setToCommaList() {
+        Set<String> set = Sets.newHashSet("a", null, "", "b");
+        
+        assertEquals("a,b", BridgeUtils.setToCommaList(set));
+        assertNull(BridgeUtils.setToCommaList(null));
+        assertNull(BridgeUtils.setToCommaList(Sets.newHashSet()));
     }
 
+    @Test(expected = UnsupportedOperationException.class)
+    public void nullsafeImmutableSet() {
+        assertNotNull(BridgeUtils.nullSafeImmutableSet(null));
+        assertNotNull(BridgeUtils.nullSafeImmutableSet(Sets.newHashSet("A")));
```

Comment:
In addition to testing they're not null, we should check that they contain the values we expect (or are empty, if expected).